### ISSUE TITLE
Validation performance

### ISSUE THREAD

**Issue by qwera123** [mingdachen/gated-attention-reader#2]
Hi!

Thanks for your pytorch implementation for this paper.

Since I get about 0.8 accuracy on training data, I want to know how about validation accuracy. Is it match to the result in the original paper?

**Comment by mingdachen** [comment]
Hi @qwera123, I did not test the model on validation set. But since you can run the code, it won't be hard for you to verify the model's performance on validation set, right?
